src/components/Nav/Nav.js

```diff
@@ -1,11 +1,53 @@
 import React from 'react';
+import './Nav.scss';
+import { Link } from 'react-router-dom';
```

Comment:
아래 순서를 참고해서 Import 순서 수정해주세요. 일반적인 convention을 따르는 이유도 있지만 순서만 잘 지켜주셔도 가독성이 좋아집니다.

- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)
```suggestion
import { Link } from 'react-router-dom';
import './Nav.scss';
```

리뷰를 남기지 않은 부분들도 찾아서 수정해주세요!